Source/Scene/ModelExperimental/ModelExperimentalSceneGraph.js

```diff
@@ -0,0 +1,156 @@
+import Axis from "../../Scene/Axis.js";
+import buildDrawCommand from "./buildDrawCommand.js";
+import defaultValue from "../../Core/defaultValue.js";
+import defined from "../../Core/defined.js";
+import Matrix4 from "../../Core/Matrix4.js";
+import ModelExperimentalSceneMeshPrimitive from "./ModelExperimentalSceneMeshPrimitive.js";
+import ModelExperimentalSceneNode from "./ModelExperimentalSceneNode.js";
+import ModelExperimentalUtility from "./ModelExperimentalUtility.js";
+import RenderResources from "./RenderResources.js";
+
+export default function ModelExperimentalSceneGraph(options) {
```

Comment:
@sanjeetsuhag do you want me to document this in my documentation sweep or do you want to handle this one since you're working on the scene graph?

Reply:
You can take this one.